src/AutoMapper/QueryableExtensions/ProjectionExpression.cs

```diff
@@ -87,7 +87,8 @@ public IQueryable<TResult> To<TResult>(IObjectDictionary parameters, params Expr
 
         internal IQueryable<TResult> To<TResult>(IObjectDictionary parameters, MemberPaths memberPathsToExpand)
         {
-            var membersToExpand = memberPathsToExpand.SelectMany(m => m).Distinct().ToArray();
+            IEnumerable<MemberInfo> selectMany = memberPathsToExpand.SelectMany(m => (m ?? new List<MemberInfo>()) as IList<MemberInfo> ?? m /*?.ToList() */);
+            MemberInfo[] membersToExpand = selectMany.Distinct().ToArray();
```

Comment:
I guess more user-friendly would be to throw a nice error, but that wouldn't help much, it's still our bug.